app/main/routes.py

```diff
@@ -0,0 +1,70 @@
+import os
+from flask import flash, json, make_response, redirect, render_template, request
+from flask_wtf.csrf import CSRFError
+from werkzeug.exceptions import HTTPException
+
+from app.main import bp
+from app.main.forms import CookiesForm
+
+
+@bp.route("/", methods=["GET", "POST"])
+def download():
+    if request.method == 'GET':
+        return render_template("download.html")
+    if request.method == 'POST':
+        # PLACEHOLDER
+        return render_template("download.html")
+
+
+@bp.route("/accessibility", methods=["GET"])
+def accessibility():
+    return render_template("accessibility.html")
+
+
+@bp.route("/cookies", methods=["GET", "POST"])
+def cookies():
+    form = CookiesForm()
+    # Default cookies policy to reject all categories of cookie
+    cookies_policy = {"functional": "no", "analytics": "no"}
+
+    if form.validate_on_submit():
+        # Update cookies policy consent from form data
+        cookies_policy["functional"] = form.functional.data
+        cookies_policy["analytics"] = form.analytics.data
+
+        # Create flash message confirmation before rendering template
+        flash("You’ve set your cookie preferences.", "success")
+
+        # Create the response so we can set the cookie before returning
+        response = make_response(render_template("cookies.html", form=form))
+
+        # Set cookies policy for one year
+        response.set_cookie("cookies_policy", json.dumps(cookies_policy), max_age=31557600)
+        return response
+    elif request.method == "GET":
```

Comment:
unnecessary elif after return (I appreciate this is an inherited lint error). Can just be if (not elif)